x-pack/plugins/security_solution/server/lib/detection_engine/rules/prepackaged_rules/persistence_google_workspace_api_access_granted_via_domain_wide_delegation_of_authority.json

```diff
@@ -50,5 +50,5 @@
   ],
   "timestamp_override": "event.ingested",
   "type": "query",
-  "version": 8
+  "version": 10
```

Comment:
Version already bumped from `6` to `8` in previous `8.2 Rules` PR, so looks to be a version jump from `6` to `10` in `8.2` without any content changes??

https://github.com/elastic/kibana/pull/129526/files#diff-3facfa2f0c2671395863e0aa9233f670f88a501985509cd5990452b62477e286R53

Reply:
Double bump is the result of manual bumps done from the following.

https://github.com/elastic/detection-rules/pull/1870/files